charts/sonatype-nexus/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: sonatype-nexus
-version: 2.8.0
+version: 2.8.1
```

Comment:
Please bump the version up to 3.0.0 as this may break existing charts.

Reply:
bumped.